### ISSUE TITLE
Merging develop into master 

### ISSUE THREAD

**Issue by duckdoom4** [LuqueDaniel/vscode-language-renpy#307]
I think it's about time we talked about merging the development branch into master.

The new syntax highlighter has most renpy features available and now includes all features that are supported on the master branch (plus more). It has been quite a while now, and it seems most of the bugs have been resolved.

I'm leaving this thread open for a while, so anyone can leave a comment before we start to work on the merge.

After the merge, we will continue to develop new features on the development branch as we did now. Master will ofcourse still be updated with bug fixes.

**Comment by LuqueDaniel** [comment]
When we do the merge we will also release version 2.2.0 right?

**Comment by FaceCrap** [comment]
Currently playing with pre-release 2.1.7 and getting malformed (erroneous parsing) entries again in the workspace file for textMateRules. 
Dunno if you want me to make this a separate issue, and/or if you need me to attach a zipped copy of a complete workspace file together with the rpy file which caused these. 

I've pasted part of one below as example

<details><summary>textMateRules snippet</summary>
<p>
<pre>				{
					"scope": "renpy.meta.color.or=#000",
					"settings": {
						"foreground": "or=#000"
					}
				},
				{
					"scope": "renpy.meta.color.lor=#2b",
					"settings": {
						"foreground": "lor=#2b"
					}
				},
				{
					"scope": "renpy.meta.color.lor=#1f",
					"settings": {
						"foreground": "lor=#1f"
					}
				},
				{
					"scope": "renpy.meta.color.ze=-4}{",
					"settings": {
						"foreground": "ze=-4}{"
					}
				},
				{
					"scope": "renpy.meta.color.hy not!",
					"settings": {
						"foreground": "hy not!"
					}
				},
				{
					"scope": "renpy.meta.color.color=#",
					"settings": {
						"foreground": "color=#"
					}
				},
</pre>
</p>
</details>

**Comment by duckdoom4** [comment]
Oh, yeah, I'm aware of that. It's due to a bug in the parser. I'm working on fixing that right now. Hope to get it all pushed today, but will have to see if I can get it all done today.
